tests/business.test.ts

```diff
@@ -40,23 +22,74 @@ import { Marqeta } from '../src';
       home: {
         address1: '106 Main St.',
         address2: '',
-        city: 'Atlanta',
-        state: 'GA',
-        postalCode: '30345',
+        city: 'Opelousas',
+        state: 'LA',
+        postalCode: '70570',
```

Comment:
Are these changes because you've already used this data? If so, then you will need a way to make data that allows for all tests... just a thought.

Reply:
It's not required to make the request parameters unique for each call.  I can leave it as is now and Marqeta will create a new Business account with the same parameters ad-nauseam returning a unique uuid token for each.